src/RamTranslationUnit.h

```diff
@@ -43,13 +43,14 @@ class RamTranslationUnit {
 
     virtual ~RamTranslationUnit() = default;
 
+    /* @brief templated method to compute/retrieve an analysis for a translation unit */
```

Comment:
Two '*'s are needed to make this a doxygen comment. e.g.
```suggestion
    /** @brief templated method to compute/retrieve an analysis for a translation unit */
```